plugins/janus_audiobridge.c

```diff
@@ -1138,6 +1141,8 @@ typedef struct janus_audiobridge_room {
 	gchar *record_file;			/* Path of the recording file */
 	FILE *recording;			/* File to record the room into */
 	gint64 record_lastupdate;	/* Time when we last updated the wav header */
+	gboolean wav_header_added;  /* if wav header is added in recording file */
```

Comment:
This property could be a `volatile int` and updated/read using GLib atomic utilities.